google-cloud-bigtable/acceptance/bigtable/cluster_test.rb

```diff
@@ -33,8 +33,8 @@
   end
 
   it "create cluster, update and delete" do
-    cluster_id = "#{$bigtable_cluster_id}2"
-    location = $bigtable_cluster_location_2
+    cluster_id = "#{bigtable_cluster_id}2"
```

Comment:
it would be nice to add an assertion for the encryption type here (GOOGLE_DEFAULT_ENCRYPTION)

Reply:
sorry, put this in the wrong place, but checking the default for non cmek clusters/tables/backups would be great

Reply:
I added assertions checking the defaults for the non-CMEK tests.